test/asynchronous/test_client.py

```diff
@@ -1938,6 +1940,7 @@ def test_sigstop_sigcont(self):
     async def _test_handshake(self, env_vars, expected_env):
         with patch.dict("os.environ", env_vars):
             metadata = copy.deepcopy(_METADATA)
+            metadata["driver"]["name"] = "AsyncPyMongo"
```

Comment:
Could we use "PyMongo-async" and add a test for DriverInfo with AsyncMongoClient?